tests/test_completion.py

```diff
@@ -467,52 +465,6 @@ def sc_app():
     return app
 
 
-def test_cmd2_subcommand_completion_single_end(sc_app):
```

Comment:
These unit tests for subcommand completion should not be deleted.  They should simply be refactored to use ``index_based_complete()`` or ``complete()`` instead of the old ``complete_subcommand()``.